pastas/fetuccinne-alfredo.md

```diff
@@ -0,0 +1,29 @@
+# Fettuccine alfredo
+
+Traditionally, this dish does not use cream, just lots of butter and parmesan. This recipe is a combination of a [Bon Appetit recipe](https://www.bonappetit.com/recipe/fettuccine-alfredo) and an [Epicurious video](https://www.youtube.com/watch?v=t81529ANEhI).
+
+## Ingredients
+- 12 ounces fettuccine or other long pasta (Preferrably fresh pasta)
+- Kosher salt
+- 1/4 cup (1/2 stick) unsalted butter
+- 3/4 cup finely grated Parmesan, plus more for serving
+- Freshly ground black pepper
+
+## Instructions
+1. In a large pot, boil water with a good amount of salt.
+
+2. When the water is boiling, add (12 ounces) pasta and cook according to the instructions on the container. Cook for the lowest amount of time required. The pasta will finish cooking in the sauce laster.
+
+3. Reserve 2 cups of the pasta cooking water when the pasta is almost done.
+
+4. Drain the pasta and set aside while the sauce is made.
+
+5. In a large skillet on medium low heat, pour 1 cup of the reseved pasta cooking water. Get it so a simmer.
+
+6. Actively whisk/mix the butter ½ to 1 tbsp at a time so the butter and the water emulsify (combine into homogenous liquid). Wait until the piece of butter is fully incorporated into the sauce before adding the next piece.
```

Comment:
I appreciate that you defined "emulsify" here, I don't think I'd know what that means.